src/components/Player.jsx

```diff
@@ -1,18 +1,98 @@
 
 export default function Player(props) {
+  const [isFlipped, setFlipped] = useState(false)
+
+  const handleClick = (e) => {
+    e.preventDefault()
+    setFlipped(!isFlipped)
+  }
 
   return (
-    <div class="w-full rounded">
+    <ReactCardFlip
+      isFlipped={isFlipped}
+      flipDirection="horizontal"
+    >
+      {/* Front of card */}
+      <div className="cursor-pointer relative" onClick={handleClick}>
+          {props.team === "wufo" ? 
+            <img 
+              src={require(`../assets/wufopics/roster/${props.name}.jpg`).default}
+              alt={`${props.name}`}
+              className="rounded"
+              style={{ objectFit: "cover", width: "300px", height: "300px" }}
+            /> :
+            <img 
+              src={require(`../assets/lawufapics/roster/${props.name}.jpg`).default}
+              alt={`${props.name}`}
+              className="rounded"
+              style={{ objectFit: "cover", width: "300px", height: "300px" }}
+            />
+          }
+          <div 
+            className="bg-black bg-opacity-50 rounded absolute right-2 bottom-2 text-white py-1 px-3 text-2xl font-semibold"
+            style={{ textShadow: '3px 2px #4C1D95' }}
+          >
+            #77 Alan
+          </div>
+      </div>
+
+      {/* Back of card */}
+      <div className="cursor-pointer relative" onClick={handleClick}>
         {props.team === "wufo" ? 
-          <img src={require(`../assets/wufopics/roster/${props.name}.jpg`).default}
-               alt={`${props.name}`}/> :
-          <img src={require(`../assets/lawufapics/roster/${props.name}.jpg`).default}
-               alt={`${props.name}`}/>
+          <img 
+            src={require(`../assets/wufopics/roster/${props.name}.jpg`).default}
+            alt={`${props.name}`}
+            className="rounded"
+            style={{ 
+              objectFit: "cover", 
+              width: "300px", 
+              height: "300px", 
+              opacity: '0.9', 
+              filter: 'brightness(40%)' 
+            }}
+          /> :
+          <img 
+            src={require(`../assets/lawufapics/roster/${props.name}.jpg`).default}
+            alt={`${props.name}`}
+            className="rounded"
+            style={{ 
+              objectFit: "cover", 
+              width: "300px", 
+              height: "300px", 
+              opacity: '0.9', 
+              filter: 'brightness(40%)' 
+            }}
+          />
         }
-        <div class="capitalize text-gray-shadow text-center">
-          {props.name}
+        <div 
+          className="absolute top-4 left-2 text-white py-1 px-3 text-2xl font-semibold"
+          style={{ textShadow: '3px 2px #4C1D95' }}
+        >
+          Alan Sun
+        </div>
+        <div 
+          className="absolute top-16 left-8 text-white py-1 px-3 text-md font-medium border-l-4 border-purple-400"
+        >
+          <span className="italic">Nickname:</span> &nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Lasuna
```

Comment:
At some point we probably wanna figure out how to align the text automatically when we dynamically populate the back of the cards

Reply:
I think with the way their positioned, they should always be aligned, tho you're right, using two divs in a row would definitely be better practice.